packages/flutter_tools/templates/ffi_plugin/src.tmpl/dart-sdk/include/dart_api.h

```diff
@@ -0,0 +1,4012 @@
+/*
```

Comment:
This is not ABI stable - we should not ship it in the template.

This seems like it should get rolled with Dart rolls. Same for dart_api_dl.

This also means that any FFI plugin that uses this API will have to be very tightly bound to a specific version of Flutter... This seems like a significant problem to me.

Reply:
(ditto for the other header/impl files from the Dart SDK)

Reply:
Removed, will address concerns with `dart_api_dl.h` in Dart SDK first and then follow up this PR.